Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -250,7 +250,7 @@ public struct CustomerCenterConfigData {
                 case .accountDetails:
                     return "Account Details"
                 case .dateWhenAppWasPurchased:
-                    return "Date when app was first purchased"
+                    return "Original Purchase Date"
```

Comment:
can we make it "date when app was first downloaded"? I feel like this still doesn't make it less confusing, especially since most apps are free

Reply:
This was actually a typo 🙃 
The copy in the backend is `"Original Download Date"`

Reply:
Since this is in the backend already, I think we can do a quick round and revisit some of them; but I would merge it like this for consistency.